templates/common/_base/units/nodeip-configuration.service.yaml

```diff
@@ -27,6 +27,9 @@ contents: |
     {{if eq .IPFamilies "IPv6" -}}
     --prefer-ipv6 \
     {{end -}}
+    {{if not (isOpenShiftManagedDefaultLB .) -}}
```

Comment:
I don't think we need this in the base service. It's not enabled for any on-prem IPI deployments, only platform None.

Reply:
ack, I'll checkout HEAD^ this one.